starter-code/src/functions-and-arrays.js

```diff
@@ -1,6 +1,27 @@
 // Find the maximum
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2){
+    return num1;
+  } else if (num1 < num2){
```

Comment:
You don't need this 'else if'. If the first number is not greater than the second one, you can return the second one, because either the second one is greater or they are equal (and then doesn't matter wich number you return):
```
function maxOfTwoNumbers(num1, num2) {
  if (num1 > num2) {
    return num1;
  } else {
    return num2;
  }
}
```